SmartDeviceLink/SDLProtocol.m

```diff
@@ -165,7 +165,7 @@ - (SDLProtocolMessage *)sdl_createStartServiceMessageWithType:(SDLServiceType)se
     switch (serviceType) {
         case SDLServiceTypeRPC: {
             // Need a different header for starting the RPC service, we get the session Id from the HU, or its the same as the RPC service's
-            header = [SDLProtocolHeader headerForVersion:1];
+            header = [SDLProtocolHeader headerForVersion:3];
```

Comment:
This is incorrect according to the protocol spec it must be a version 1 header.

Reply:
When sent with headerVersion 1 SYNC sends a NACK.  According the Luxoft, it should be 3 or above, this result in ACK in testing

Reply:
I'm pretty sure that this will drop support for protocol v1 and v2 head units.

Reply:
In testing this always sends an ACK with encryption = 0 (failed to start due to encryption bit set to 0 in ACK)